src/docs/metis.mdx

```diff
@@ -0,0 +1,163 @@
+                    tooltip: 'The gas cost of proving and executing a message on Ethereum after being sent from Metis',
+                    value: '740 000 L1 gas',
+                },
+            ],
+        },
+    ]} />
+
+</Section>
+
+<Section title="OPCODEs">
+
+    > ⚠️ if you're using solidity 0.8.9 or above, the default evm version is London, and it's not supported by Metis, you should set evm version to berlin.
+
+    <Legend />
+
+    The following table encapsulates EVM OPCODE differences of the Rollup compared to the canonical Ethereum L1 implementation.
+
+    | Opcode | Name | Solidity Equivalent | Rollup Behaviour | Ethereum L1 Behaviour |
+    | :----- | :--- | :-------------------| :--------------- | :-------------------- |
+    | 3A | GASPRICE | `blockhash.gasPrice` | returns constant `0` | Get price of gas in current environment <Modified /> |
+    | 40 | BLOCKHASH | `blockhash(block.number)` | returns constant `0x0000000000000000000000000000000000000000000000000000000000000000` | Get the hash of one of the 256 most recent complete blocks <Unsupported /> |
+    | 41 | COINBASE | `block.coinbase` | returns constant `0x4200000000000000000000000000000000000011` | Get the L1 block’s beneficiary address <Modified /> |
+    | 42 | TIMESTAMP | `block.timestamp` | Returns timestamp up to 60 seconds in the past. | Timestamp of the L1 block <Modified /> |
+    | 43 | NUMBER | `block.number` | L2 block number | Gets the L1 block number <Modified /> |
+    | 44 | PREVRANDAO | `block.prevrandao` | Returns constant `0` | Get the output of the randomness beacon provided by the beacon chain <Modified /> |
+    | 48 | BASEFEE | `block.basefee` | Undefined | Get the base fee <Unsupported /> |
+    | 5F | PUSH0 |  | Not supported | Places value 0 on the stack <Unsupported /> |
+    
+</Section>
+
+<Section title="Precompiled Contracts">
+
+    All Precompiled Contracts defined in the canonical Ethereum L1 implementation have the same behaviour on the Rollup.
+
+</Section>
+
+<Section title="System Contracts">
+
+    The Rollup has introduced the following system contracts.
+
+    | Address | Name | Description |
+    | :--- | :--- | :--- |
+    | <Copy value="0x6A1DB7d799FBA381F2a518cA859ED30cB8E1d41a" label="0x6A1...1a" /> | <Reference label="CanonicalTransaction" url="https://github.com/MetisProtocol/mvm/blob/develop/packages/contracts/contracts/MVM/MVM_CanonicalTransaction.sol" /> | The Canonical Transaction Chain (CTC) contract is an append-only log of transactions which must be applied to the rollup state. |
+    | <Copy value="0xf209815E595Cdf3ed0aAF9665b1772e608AB9380" label="0xf20...80" /> | <Reference label="StateCommitmentChain" url="https://github.com/MetisProtocol/mvm/blob/develop/packages/contracts/contracts/L1/rollup/StateCommitmentChain.sol" /> | The State Commitment Chain (SCC) contract contains a list of proposed state roots which Proposers assert to be a result of each transaction in the Canonical Transaction Chain (CTC). |
```

Comment:
Are there any user facing functions and features that can be used by non-system addresses? If no I would consider removing these addresses from the "system contracts" table as we do for other rollups